extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/caffeine/CaffeineCache.java

```diff
@@ -31,11 +31,7 @@
     public CaffeineCache(CaffeineCacheInfo cacheInfo) {
         this.name = cacheInfo.name;
         Caffeine<Object, Object> builder = Caffeine.newBuilder();
-        /*
-         * The following line is a workaround for a GraalVM issue: https://github.com/oracle/graal/issues/1524
-         * TODO: Remove it as soon as a Quarkus release depends on GraalVM 19.3.0 or higher.
-         */
-        builder.executor(task -> ForkJoinPool.commonPool().execute(task));
```

Comment:
Good catch! This should have been removed in #6792 when we dropped the GraalVM 19.2.1 support.

The build failed because you need to remove the `ForkJoinPool` import. Could you fix that please?

Reply:
Sorry, fixed

Reply:
Thanks! Let's see if CI is happy with this in native mode now.

Reply:
Should be, I went through the same issue with http://github.com/quarkusio/quarkus/pull/6758 :)